### PR TITLE
Bug 1836818: Add validation of csisnapshotcontroller

### PR DESCRIPTION
**Pull request by jsafrane**
- metadata.name must be "cluster"
- spec.logLevel and spec.operatorLogLevel must be Normal, Debug, Trace or TraceAll

### CODE REVIEW

**Comment by jsafrane** [comment]
cc @openshift/storage

File: manifests/02_operator_cr.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ metadata:
   annotations:
     release.openshift.io/create-only: "true"
 spec:
-  managementState: Managed
\ No newline at end of file
+  managementState: Managed
```
Comment (bertinatto) [inline manifests/02_operator_cr.yaml:8]: Was this intentional?

**Pull request merged**